modules/caddyhttp/opentelemetry/README.md

```diff
@@ -0,0 +1,59 @@
+# OpenTelemetry module
+
+This module provides integration with OpenTelemetry tracing facilities. It is implemented
+as `caddyhttp.MiddlewareHandler` and can be chained into a list of other handlers.
+
+When enabled, it propagates an existing tracing context or will init a new one otherwise.
+
+It is based on `https://github.com/open-telemetry/opentelemetry-go`.
+
+## Configuration
+
+### Environment variables
+
+It can be configured using environment variables defined
+by [spec](https://github.com/open-telemetry/opentelemetry-specification).
```

Comment:
Can we link directly to the environment variables spec?

```suggestion
It can be configured using the environment variables defined
by the [OpenTelemetry Environment Variable Specification](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/sdk-environment-variables.md).
```

Reply:
Perfectly makes sense to me. Good point! 👍